### PR TITLE
add branch name in the helper template

### PR DESCRIPTION
**Pull request by kay0u**
##  Solution

Add the current branch during doc generation. Should we use the commit hash instead to avoid "desync"?

We can see the result of the job here: https://yunohost.gitlab.io/-/yunohost/-/jobs/874635832/artifacts/doc/helpers.html

## PR Status

...

## How to test

...

### CODE REVIEW

**Comment by alexAubin** [comment]
> Should we use the commit hash instead to avoid "desync"?

Hmmmyes maybe ?

I guess it depends on the intention : do we want to keep the doc in sync with the very-latest stuff in dev (in that case keeping the branch name dev is okay) or if we only push the doc change when releasing a stable ? (I haven't 100% followed the other branch for autopush from gitlab builds)

**Comment by kay0u** [comment]
I think we want to keep the stable version of the doc, therefore I did this https://github.com/YunoHost/yunohost/pull/1080 which should create a PR on each new YunoHost version.

**Comment by alexAubin** [comment]
So yup I would say we wanna point to the commit ideally :+1:

File: doc/generate_helper_doc.py [hunk 1]
```diff
@@ -3,10 +3,19 @@
 import os
 import glob
 import datetime
+import subprocess
 
+def get_current_git_branch():
+    p = subprocess.Popen("git branch -a --contains | grep remote | cut -d'/' -f3 ", shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```
Comment (kay0u) [inline doc/generate_helper_doc.py:9]:
```suggestion
    p = subprocess.Popen("git rev-parse --verify HEAD", shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

**Pull request merged**